pure_pursuit_wrapper/launch/pure_pursuit_wrapper.launch

```diff
@@ -10,6 +10,7 @@
             <arg name="is_linear_interpolation" value="True"/>
             <arg name="publishes_for_steering_robot" value="True"/>
         </include>
+        <rosparam command="load" file="$(arg vehicle_calibration_dir)/pure_pursuit/calibration.yaml"/>
```

Comment:
There is a good chance these parameters will not be used by pure_pursuit because they are loaded after it is launched. 
There is also a publisher in guidance.launch that needs to be removed so this is not overwritten. Misheel can point you to it.